website/templates/project/project_base.mako

```diff
@@ -195,7 +195,8 @@
                 api: nodeApiUrl,
                 web: ${ node['url'] | sjson, n },
                 update: ${ node['update_url'] | sjson, n },
-                waterbutler: ${node['waterbutler_url']| sjson, n }
+                waterbutler: ${node['waterbutler_url']| sjson, n },
+                mfr: ${ node['mfr_url'] if node['mfr_url'].endswith('/') else node['mfr_url'] + '/' | sjson, n }
```

Comment:
Is there a reason we need to handle this?

In any case, this can be simplified to

```python
node['mfr_url'].rstrip('/') + '/'
```

Reply:
Oh I see now--you just moved existing code. OK no biggie, then.